### PR TITLE
updates for social page in the new folder structure 

### PR DESCRIPTION
**Pull request by akintola4**
# moved all code for questPage to socials
# deleted older questPage

### CODE REVIEW

File: frontend/src/app/app/quests/socials/page.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// app/QuestPage/page.tsx
+'use client'
```
Comment (Dprof-in-tech) [inline frontend/src/app/app/quests/socials/page.tsx:2]: eslint errors and it doesnt work

File: frontend/src/app/app/quests/socials/layout.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import NavMain from './components/NavMain'
 import { ReactNode, useContext } from "react";
 import { usePathname } from 'next/navigation'
```
Comment (Dprof-in-tech) [inline frontend/src/app/app/quests/socials/layout.tsx:8]: why do you need a layout file? look at the structure for the profile/page.tsx and see how the sidebar was implemented there.

**Review by Dprof-in-tech — CHANGES_REQUESTED** [review]
Also, move all your components into the components folder /quests/component. and make sure the code is running before pushing a new commit into this PR

**Comment by DanielEmmanuel1** [comment]
Looks good to me @akintola4

**Comment by akintola4** [comment]
Thank you sir

**Pull request merged**